source/dbo/Stored Procedures/Power BI Reporting/uspGenerateNtbsCaseRecord.sql

```diff
@@ -32,6 +32,16 @@ BEGIN TRY
 		FROM venues
 		GROUP BY NotificationId;
 
+	WITH firstTransferOut AS
+	(SELECT t.NotificationId, TbServiceCode, tbs.[Name]
+		FROM (SELECT DISTINCT NotificationId FROM [$(NTBS)].dbo.TreatmentEvent) te
+		CROSS APPLY (SELECT TOP 1 *
+			FROM [$(NTBS)].dbo.TreatmentEvent te2
+			WHERE te2.NotificationId = te.NotificationId AND TreatmentEventType = 'TransferOut'
+			ORDER BY EventDate) t
+		JOIN [$(NTBS)].ReferenceData.TbService tbs ON tbs.Code = t.TbServiceCode)
+	SELECT * INTO #TempTransfers FROM firstTransferOut;
```

Comment:
The "notifying" fields are really just "TB service responsible at 0 months" so I think we should calculate them all in the same file, and use similar logic for all